### DISCUSSION TITLE
Dependabot says it created some PRs but it doesn't

### DISCUSSION THREAD

**Discussion by llucax** [community/community#65844] [Code Security]
### Select Topic Area

Bug

### Body

Hi, I noticed one project is not getting dependabot PRs. Checking at the run logs, dependabot says it is creating PRs, but the PRs are just not there (neither for pip or github actions):

* https://github.com/frequenz-floss/frequenz-repo-config-python/network/updates/717167531
* https://github.com/frequenz-floss/frequenz-repo-config-python/network/updates/717167130
* https://github.com/frequenz-floss/frequenz-repo-config-python/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc
* https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/.github/dependabot.yml

I tried re-running dependabot and it is still the same. I'm not sure if this can be a misconfiguration or if it is a bug.

Thanks!

**Comment by carogalvin** [comment]
Hi, I recommend opening an issue in the Dependabot core repo: https://github.com/dependabot/dependabot-core/issues/new?assignees=&labels=T%3A+bug+%F0%9F%90%9E&projects=&template=bug-report.yml

**Comment by llucax** [comment]
Hi @carogalvin, thanks for the suggestion.

I reported this to GitHub support and they identified the issue being that the repository has a file [`.gitmodules`](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/cookiecutter/%7B%7Bcookiecutter.github_repo_name%7D%7D/.gitmodules) that doesn't valid format for a git modules file (it is a template). This is clearly a bug, an error parsing a git submodules file, specially when dependabot is not handling git as a package manager, should not stop other updates, it should  be just ignored.
